src/content/docs/apis/nerdgraph/tutorials/provisions-your-subscriptions-nerdgraph.mdx

```diff
@@ -3,39 +3,34 @@ title: "NerdGraph tutorial: Partner and reseller subscriptions"
 metaDescription: 'For New Relic partners: create subscriptions for your accounts.'
 ---
 
-<Callout variant="important">
-  The Provisioning API is only available for [Partner accounts](/docs/new-relic-partnerships/partner-integration-guide/getting-started/partner-integration-requirements) on our [original pricing model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#pricing-plans). 
-</Callout>
 
-The Provisioning NerdGraph API allows you, as a [New Relic partner](/docs/new-relic-partnerships/partner-integration-guide/getting-started/partner-integration-requirements) to create subscriptions for your accounts containing a more extensive range of New Relic products than the [Partner API](/docs/new-relic-partnerships/partner-integration-guide/partner-account-maintenance/partner-api).
+The subscription provisioning functionality of our NerdGraph API allows you, as a [New Relic partner](/docs/new-relic-partnerships/partner-integration-guide/getting-started/partner-integration-requirements), to create subscriptions for your accounts. And it gives you access to a more extensive range of New Relic products than the [Partner API](/docs/new-relic-partnerships/partner-integration-guide/partner-account-maintenance/partner-api).
 
-The Provisioning API and Partnership API are compatible when it comes to the products supported by both (APM, Mobile, Insights, Browser, Synthetics, and Infrastructure). If you use the Provisioning API to create a subscription for any of the newer products, you will no longer be able to update the account using the Partnership API V2.
+## Requirements [#requirements] 
 
-The Provisioning API is a GraphQL mutation field named `organizationProvisioningUpdatePartnerSubscription`. To learn more about how to use this mutation, see [Introduction to NerdGraph](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph).
+* You should **not** use NerdGraph to adjust your subscription provisioning unless specifically instructed by your New Relic representative.
+* Only available for [Partner accounts](/docs/new-relic-partnerships/partner-integration-guide/getting-started/partner-integration-requirements) that are on our [original pricing model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#pricing-plans). 
 
-## Get started: endpoint and API key [#endpoint]
+NerdGraph's provisioning functionality and the [Partnership API](/docs/new-relic-partnerships/partner-integration-guide/partner-account-maintenance/partner-api) are compatible when it comes to the products supported by both (APM, Mobile, Insights, Browser, Synthetics, and Infrastructure). If you use NerdGraph to create a subscription for any of the newer products, you'll no longer be able to update the account using the Partnership API.
```

Comment:
we no longer refer to these as products or capitalize them. see style guide https://docs.newrelic.com/docs/style-guide/capitalization/product-capability-feature-usage/

Reply:
right, although i wasn't reviewing for this stuff and there is a ton of that in these docs (and in some sense it's not bad as this is referring to original pricing model, which still has those product classifications)